File: myulib-core/src/main/java/com/myudog/myulib/api/MyuVFX.java
package com.myudog.myulib.api;

import com.myudog.myulib.api.core.floating.IFloatingObject;
import com.myudog.myulib.internal.entity.ItemDisplayObject;
import net.minecraft.world.item.ItemStack;
import net.minecraft.server.level.ServerLevel;



public final class MyuVFX {
	private MyuVFX() {
	}

	public static IFloatingObject createItemObject(ServerLevel Level, ItemStack itemStack) {
		return new ItemDisplayObject(Level, itemStack);
	}
}

File: src/main/kotlin/com/myudog/myulib/api/MyuVFX.kt
package com.myudog.myulib.api

import com.myudog.myulib.api.`object`.IFloatingObject
import com.myudog.myulib.internal.entity.ItemDisplayObject
import net.minecraft.item.ItemStack
import net.minecraft.server.world.ServerWorld

object MyuVFX {
    /**
     * 創建一個懸浮的物品模型
     */
    fun createItemObject(world: ServerWorld, itemStack: ItemStack): IFloatingObject {
        return ItemDisplayObject(world, itemStack)
    }
}